spec/avrolution/configuration_spec.rb

```diff
@@ -0,0 +1,95 @@
+describe Avrolution, "configuration" do
+  describe "#compatibility_schema_registry_url" do
+    let(:configured_value) { 'http://static.example.com' }
+
+    subject(:url) do
+      described_class.compatibility_schema_registry_url
+    end
+
+    before do
+      Avrolution.compatibility_schema_registry_url = configured_value
+    end
+
+    it "returns the configured value" do
+      expect(url).to eq(configured_value)
+    end
+
+    context "when the configured value is a Proc" do
+      let(:dynamic_value) { 'http://dynamic.example.com' }
+      let(:configured_value) do
+        -> { dynamic_value }
+      end
+
+      it "returns the result of the proc" do
+        expect(url).to eq(dynamic_value)
+      end
+    end
+
+    context "when a value is not configured" do
+      let(:configured_value) { nil }
+
+      it "raises an error" do
+        expect { url }.to raise_error('compatibility_schema_registry_url must be set')
+      end
+    end
+
+    context "when the environment variable is set" do
```

Comment:
There is an assigned value on line 10 (https://github.com/salsify/avrolution/pull/6/files#diff-b357f6cd5e7a00c7f6afb2b01e6ff4dbR10) that this takes precedence over.